ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -133,7 +136,22 @@ type opennebula_ceph_datastore = {
     "rbd_format"                ? long(1..2)
 };
 
-@{ type for an opennebula datastore. Defaults to a ceph datastore (ds_mad is ceph) @}
+@documentation{ 
+type for vnet ars specific attributes. 
+type and size are mandatory 
+}
+type opennebula_ar = {
+    "type"                      : string with match(SELF, "^(IP4|IP6|IP4_6|ETHER)$")
+    "ip"                        ? type_ipv4
```

Comment:
opennebula ignores the values that are not used, so now is quite safe and flexible but in any case I can try different possibilities in our testbed and see if we get some issue. If so yes, we can include a validation rule to be more strict with the pool definition.